test/l2/dai.ts

```diff
@@ -26,6 +24,16 @@ describe('Counter', () => {
       name.should.equal('Dai Stablecoin')
     })
 
+    it('returns the symbol', async () => {
+      const name = await dai.symbol()
+      name.should.equal('DAI')
```

Comment:
can we migrate everywhere to `expect()...` syntax and drop import `should` entirely?